modules/system/console/OctoberPasswd.php

```diff
@@ -0,0 +1,129 @@
+<?php namespace System\Console;
+
+use Str;
+use Backend\Models\User;
+use Illuminate\Console\Command;
+use Illuminate\Database\Eloquent\ModelNotFoundException;
+use Symfony\Component\Console\Input\InputArgument;
+use Symfony\Component\Console\Question\Question;
+
+/**
+ * Console command to change the password of a Backend user via CLI.
+ *
+ * This command may only be run through CLI.
+ *
+ * @package october\system
+ */
+class OctoberPasswd extends Command
+{
+    /**
+     * The console command name.
+     *
+     * @var string
+     */
+    protected $name = 'october:passwd';
+
+    /**
+     * The console command description.
+     *
+     * @var string
+     */
+    protected $description = 'Change the password of a Backend user.';
+
+    /**
+     * Was the password automatically generated?
+     *
+     * @var bool
+     */
+    protected $generatedPassword = false;
+
+    /**
+     * Execute the console command.
+     */
+    public function handle()
+    {
+        // Must be in CLI environment
+        if (php_sapi_name() !== 'cli') {
+            $this->error('This command may only be run through the command-line.');
+            exit(1);
+        }
```

Comment:
Why is this command restricted to the CLI? Additionally, I would prefer the use of `App::runningInConsole()` if we do continue to restrict it in this way

Reply:
When I saw this I assumed it to be an extra security precaution to prevent the `Artisan::call()` from somehow programatically calling this and resetting an account. You never know what people will do and this is a "better safe, than sorry" type check

Reply:
@daftspunk is correct - I didn't want to allow people to use `Artisan::call()` and sneak it a plugin or something. People who have access to the CLI are very safe to assume that they are trusted, so I felt it was best to restrict it to there.

Reply:
While this is nice in theory there is actually zero security benefit in practice to doing this since the there's absolutely nothing stopping a plugin from doing the exact same thing that this command does internally. Hence I don't think there's any reason to include that check. The only reason I can think is if you wanted to protect against a theoretical plugin that allowed any backend users to call artisan commandos through the backend, but such a plugin has a whole host of other reasons why it shouldn't exist.

Reply:
From the other point of view, I don't think there's any legitimate reason to make it available outside of CLI. But I'll remove it shortly.